WordPressKit/WordPressKit/WordPressOrgXMLRPCValidator.swift

```diff
@@ -193,7 +193,15 @@ open class WordPressOrgXMLRPCValidator: NSObject {
                         return
                     }
                 }
-                failure(error)
+                
+                switch httpResponse?.statusCode {
+                case .some(403):
```

Comment:
Instead of hardcoding `403` here, can we do `.some(WordPressOrgXMLRPCValidatorError.forbidden.rawValue)`?